src/lib/validation.ts

```diff
@@ -12,6 +12,7 @@ export interface CheckIntOptions {
 
 const ENV_VAR_KEY_REGEX = /^[a-zA-Z_][a-zA-Z0-9_]*$/;
 const LABEL_NAME_REGEX = /^[a-zA-Z][a-zA-Z0-9\.\-]*$/;
+const NUMERALS_REGEX = /^-?[0-9]+\.?0*$/; // Allows trailing 0 decimals
```

Comment:
Why not `\d` ?

Reply:
`\d` will match any unicode-defined digit. Since there are many notations for number systems, I think it's safest to explicitly have what values we want instead of thinking of all the edge cases possible.